packages/build/package.json

```diff
@@ -33,15 +33,20 @@
     "lb-clean": "./bin/run-clean.js"
   },
   "scripts": {
+    "build:apidocs": "lb-apidocs",
     "lint": "npm run prettier:check",
     "lint:fix": "npm run prettier:fix",
+    "prepare": "npm run build:apidocs",
     "prettier:cli": "node bin/run-prettier \"bin/**/*.js\"",
     "prettier:check": "npm run prettier:cli -- -l",
     "prettier:fix": "npm run prettier:cli -- --write",
     "test": "npm run mocha",
     "mocha": "node bin/select-dist mocha --timeout 30000 --opts test/mocha.opts \"test/integration/*.js\"",
     "posttest": "npm run lint"
   },
+  "files": [
+    "api-docs"
```

Comment:
IIUC, this means that `npm publish` will include only api-docs files, but not the actual build helpers! See https://docs.npmjs.com/files/package.json#files

I think you should either revert this addition of a "file" field, or else make sure all important files/directories are listed in "files".